### DISCUSSION TITLE
80s 8-bit connected to the emulator

### DISCUSSION THREAD

**Discussion by SirNibble** [prirun/p50em#11] [General]
Taken a few days ago - my '80s C64s conncted to my Pi4 via a WiFi Modem.

![20210115_164504](https://user-images.githubusercontent.com/77676973/105979585-c3679c00-608b-11eb-94af-4fb2eec91863.jpg)

Also had my BBC talking to the emulator through it's RS423 -> USB -> PC -> bridge ethernet -> Pi4

![vlcsnap-2021-01-27-10h42m30s548](https://user-images.githubusercontent.com/77676973/105980264-8b148d80-608c-11eb-835b-259f409971a3.png)

Utterly pointless, but a rather fun sense of achievement!

**Comment by drboone** [comment]
Nice!  Back then, I used my Osborne 1 and my SB-180 from home, and a Televideo TPC-1 from work, as terminals, in addition to actual "dumb" terminals.

Note that the emulator can treat a host serial port (e.g. USB serial dongle) as an AMLC line.

**Comment by SirNibble** [comment]
Ooo! I didn't know that about the serial port. Not an immediate use case at the mo, but does make me wonder if that could be used with the Spooler.

**Comment by drboone** [comment]
I think that's part of the reason Jim built that feature.  I've also used the outbound socket connection support to spool to my LaserJet's network card.
